src/IRsend.h

```diff
@@ -204,7 +204,7 @@ class IRsend {
   static uint16_t defaultBits(const decode_type_t protocol);
   bool send(const decode_type_t type, const uint64_t data,
             const uint16_t nbits, const uint16_t repeat = kNoRepeat);
-  bool send(const decode_type_t type, const uint8_t state[],
+  bool send(const decode_type_t type, const unsigned char *state,
```

Comment:
Why switch to unsigned char? Code style guidelines says otherwise?

Reply:
Fixed.
Why? Because I was blindly playing whack-a-mole when doing the Doxygen stuff and wasn't thinking at the time. :-) Now, I just blindly copied the diffs for "code" changes from my other api_doco branch without thinking again. Sense a theme? :)

Reply:
We all have that mode at times, and it is good to have that as well.